ui/src/main/resources/PhenoTips/SimilarCases.xml

```diff
@@ -54,7 +54,17 @@
   "resultsCount" : $matches.size(),
   "results" : [
 #foreach ($p in $matches)
-  $p.toJSON()#if ($foreach.hasNext()),
+   #set ($match = $p.toJSON())
```

Comment:
I was actually thinking it would be better to use Gene class from PT-3072 PR. But it involves refactoring gene code in patient-network (which was planned to be discussed), so I can just quick fix it in Java for now if urgently needed.